extensions/amp-story-subscriptions/0.1/amp-story-subscriptions.css

```diff
@@ -90,9 +91,9 @@ iframe.swg-dialog {
   font-weight: 400 !important;
 }
 
-/* TODO(#38112): confirm with UX about if there's better animation to be used. */
 .i-amphtml-story-subscriptions-dialog-banner-button.i-amphtml-story-subscriptions-dialog-banner-button-visible {
   opacity: 1 !important;
+  transform: translate(0.5em) !important;
 }
 
 .i-amphtml-story-subscriptions-dialog-banner-button:after {
```

Comment:
In the example I posted this element is also transitioned by `.1em`.
it's subtle but ads polish to the animation.